frontend/src/app/register-form/register-form.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, Validators, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-register-form',
+  templateUrl: './register-form.component.html',
+  styleUrls: ['./register-form.component.scss'],
+})
+export class RegisterFormComponent implements OnInit {
+  registerForm: FormGroup;
+
+  constructor(private fb: FormBuilder) {
+    this.registerForm = this.fb.group({
+      name: ['', [Validators.required, Validators.minLength(3)]],
+      email: ['', [Validators.required, Validators.email]],
+      password: ['', [Validators.required, Validators.minLength(3)]],
+      repeatPassword: ['', Validators.required],
+    });
```

Comment:
You can initialise form inline, above the constructor:
```ts
registerForm: FormGroup = this.fb.group({
      name: ['', [Validators.required, Validators.minLength(3)]],
      email: ['', [Validators.required, Validators.email]],
      password: ['', [Validators.required, Validators.minLength(3)]],
      repeatPassword: ['', Validators.required],
    });
```

Benefit? You could remove `*ngIf=registerForm` from the HTML template.

Reply:
Thanks for pointing this out ;) Already corrected